pkg/metaserver/external/manager_linux.go

```diff
@@ -92,3 +95,10 @@ func (m *externalManagerImpl) setComponentImplementation(setter func()) {
 
 	setter()
 }
+
+func (m *externalManagerImpl) AdjustNumaMB(node int, avgMB, quota uint64, action mbm.MB_CONTROL_ACTION) error {
+	if mbw.MBAdjuster == nil {
+		return errors.New("low level mbw not enabled")
+	}
+	return mbw.MBAdjuster.AdjustNumaMB(node, avgMB, quota, action)
```

Comment:
why not implement MBAdjuster as a property of externalManagerImpl, just like NetworkManager or CgroupIDManager

Reply:
good advice. will update code as suggested.